### PR TITLE
Move opset to ov

### PR DESCRIPTION
**Pull request by ilyachur**
### Details:
 - Moved opsets to ov namespace
 - ngraph::OpSet inherited from ov::OpSet to add some legacy methods.
 - Depends on https://github.com/openvinotoolkit/openvino/pull/7365

### CODE REVIEW

File: ngraph/core/include/openvino/opsets/opset.hpp [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+#pragma once
+
+#include <locale>
+#include <map>
+#include <mutex>
+#include <set>
+#include <utility>
+
+#include "ngraph/factory.hpp"
```
Comment (ilya-lavrenov) [inline ngraph/core/include/openvino/opsets/opset.hpp:13]: will it changed later?

**Review by ilya-lavrenov — APPROVED** [review]

File: ngraph/core/include/openvino/opsets/opset.hpp (hunk 1 above)
Comment (ilyachur) [reply ngraph/core/include/openvino/opsets/opset.hpp:13]: Yes we will remove it with old API

**Comment by ilyachur** [comment]
@mitruska @tomdol Please have a look

**Review by mitruska — APPROVED** [review]

File: ngraph/core/include/ngraph/opsets/opset.hpp [hunk 2]
```diff
@@ -12,99 +12,36 @@
 #include "ngraph/factory.hpp"
 #include "ngraph/ngraph_visibility.hpp"
 #include "ngraph/node.hpp"
+#include "openvino/opsets/opset.hpp"
 
 namespace ngraph {
 /// \brief Run-time opset information
-class NGRAPH_API OpSet {
+class NGRAPH_API OpSet : public ov::OpSet {
```
Comment (tomdol) [inline ngraph/core/include/ngraph/opsets/opset.hpp:19]: Can you comment why do we have inheritance here? Is the legacy opset usage different from ov::OpSet?

**Review by tomdol — APPROVED** [review]

File: ngraph/core/include/ngraph/opsets/opset.hpp (hunk 2 above)
Comment (ilyachur) [reply ngraph/core/include/ngraph/opsets/opset.hpp:19]: Yes, the legacy API has several methods which work with Factory (which we don't want move to new public API).
This inheritance allows to keep methods which use Facroty in old API and hides this Facroty in new API.

File: ngraph/core/include/openvino/opsets/opset.hpp (hunk 1 above)
Comment (ilya-lavrenov) [reply ngraph/core/include/openvino/opsets/opset.hpp:13]: Do you mean we will remove it before 2022.1 is released?

**Pull request merged**